base/subarray.jl

```diff
@@ -211,10 +211,6 @@ function getindex{T,N}(V::FastSubArray{T,N}, I::Vararg{Real,N})
     r
 end
 
-# Nonscalar indexing just copies a view
-getindex{T,N}(V::SubArray{T,N}, i::ViewIndex) = (@_propagate_inbounds_meta; copy(slice(V, i)))
-getindex{T,N}(V::SubArray{T,N}, I::Vararg{ViewIndex,N}) = (@_propagate_inbounds_meta; copy(slice(V, I...)))
-
```

Comment:
Hah, I love it.  I think this means that you can delete a few more methods, too.  See the comment at line 182 (not sure if this link will work: https://github.com/JuliaLang/julia/pull/16260/commits/19a8b412803c53635ff8f6481dc8a525de469015#diff-c8c9f86af939c6f462b38975999f448cR182 ).